app/scripts/controllers/gene.js

```diff
@@ -3251,6 +3251,7 @@ angular.module('oncokbApp')
                     var panel = document.getElementById(uuid);
                     if (panel.style.display === "none") {
                         panel.style.display = "block";
+                        $scope.$broadcast("elastic:adjust");
```

Comment:
I thought once we change back to ng-if, this is no longer needed

Reply:
This is for toggle mutation tab